project-base/app/src/DataFixtures/Demo/ArticleDataFixture.php

```diff
@@ -81,12 +81,15 @@ private function getDataForArticles(DomainConfig $domainConfig): array
             ], [
                 self::ATTRIBUTE_PLAIN_NAME_KEY => 'For press',
                 self::ATTRIBUTE_NAME_KEY => t('For press', [], Translator::DATA_FIXTURES_TRANSLATION_DOMAIN, $locale),
-                self::ATTRIBUTE_TEXT_KEY => t(
+                self::ATTRIBUTE_TEXT_KEY => '<div class="gjs-text-with-image">
+<div data-image-position="left" data-image-type="outside-layout" class="gjs-text-with-image-inner gjs-text-with-image-float-left gjs-text-with-image-type-outside-layout">
+<img data-image-position="left" src="' . $domainConfig->getUrl() . '/content/images/blogArticle/600.jpg" class="image">
+<div class="gjs-text-ckeditor text" data-gjs-type="text">' . t(
                     'Morbi posuere mauris dolor, quis accumsan dolor ullamcorper eget. Phasellus at elementum magna, et pretium neque. Praesent tristique lorem mi, eget varius quam aliquam eget. Vivamus ultrices interdum nisi, sed placerat lectus fermentum non. Phasellus ac quam vitae nisi aliquam vestibulum. Sed rhoncus tortor a arcu sagittis placerat. Nulla lectus nunc, ultrices ac faucibus sed, accumsan nec diam. Nam auctor neque quis tincidunt tempus. Nunc eget risus tristique, lobortis metus vitae, pellentesque leo. Vivamus placerat turpis ac dolor vehicula tincidunt. Sed venenatis, ante id ultrices convallis, lacus elit porttitor dolor, non porta risus ipsum ac justo. Integer id pretium quam, id placerat nulla.',
                     [],
                     Translator::DATA_FIXTURES_TRANSLATION_DOMAIN,
                     $locale,
-                ) . '|||[gjc-comp-ProductList&#61;9177759,91765223,9176522,91765782,8981537,8981538,9176578]|||',
+                ) . '</div></div></div>|||[gjc-comp-ProductList&#61;9177759,91765223,9176522,91765782,8981537,8981538,9176578]|||',
```

Comment:
Can you coordinate this PR with https://github.com/shopsys/shopsys/pull/3908/files ?

There should be fixed a product list display (currently it renders like this in grapesjs)
![Snímek obrazovky 2025-05-23 v 15 32 48](https://github.com/user-attachments/assets/20c58e71-a7c9-411d-a542-4d69a6bc8f03)

and more content is added